app/views/budgets/investments/_investment_show.html.erb

```diff
@@ -51,12 +51,6 @@
         </div>
       <% end %>
 
-      <% if investment.external_url.present? %>
```

Comment:
But... we want existing budgets with external_url's but no documents to still show those values right?

Reply:
Yes, we want to disallow this input field, but maintain the existing links 😌